osbuild/util/parsing.py

```diff
@@ -31,3 +33,72 @@ def parse_size(s: str) -> Union[int, str]:
                 return "unlimited"
 
     raise TypeError(f"invalid size value: '{s}'")
+
+
+def parse_mount(url: ParseResult, args: Dict) -> os.PathLike:
```

Comment:
Now that this is a shared utility function I wonder if we can find a less generic name. Maybe something like `find_root_for_mount_url()` ? Happy about more ideas, maybe @achilleas-k has some good ones. And once we have a pattern let's use it for parse_input as well.

Reply:
Your suggestion is nicely descriptive but maybe a bit too long.  Perhaps it's not important to mention `url` there and it could just be `find_root_for_mount()`, or better yet `find_mount_root()`?